routes/web.php

```diff
@@ -17,6 +19,22 @@
     return view('welcome');
 });
 
+//
+Route::get('movies', [MovieController::class, 'index'])->name('movies.index');
+
+Route::get('movies/create', [MovieController::class, 'create'])->name('movies.create');
+
+Route::get('movies/{movie}/edit', [MovieController::class, 'edit'])->name('movies.edit');
+
+Route::get('movies/{movie}', [MovieController::class, 'show'])->name('movies.show');
+
+Route::post('movies', [MovieController::class, 'store'])->name('movies.store');
+
+Route::put('movies/{movie}', [MovieController::class, 'update'])->name('movies.update');
+
+Route::delete('movies/{movie}', [MovieController::class, 'destroy'])->name('movies.destroy');
```

Comment:
As I mentioned in one of the comments above, I think that we should check which of these routes can be accessed by a user and which can be accessed by a guest. Take a look at the Laravel docs and check for `Route::group`, you can group your routes and use the same middleware. In this case, you can separate them in guests and auth.

Reply:
I could not find the Route::group that you made mention of. The closest that I could find in the laravel documentation is Route::middleware. But doing this still manage to seperate some of the routes between required auth routes and regular 'guest' routes

Reply:
https://laravel.com/docs/8.x/routing Check the docs :D they have some examples of a route group there

Reply:
Yes, that is correct @FinnMaladay, to use a middleware in a group you do need to use the `Route::middleware()` :D you are looking in the right place. And the example they have can be applicable in your scenario too. 

The `Route::group` is used in that example as well. Just look a bit closer and you will see that group() is chained right after the middleware().